### PR TITLE
Fix initial values initialization

### PR DESCRIPTION
**Pull request by iRet**
Issue  #17 

Addresses initial values issue for volume and pitch faders.
EQ and gain knobs.

### CODE REVIEW

**Comment by Lykos153** [comment]
Thanks a lot for contributing these pull requests! I'm very happy to see some activity here again! I cannot currently review any of this, because due to a lack of time I'm not actively doing anything with Mixxx and thus don't have the Roland here with me now. I'm actually inclined to give you write access to the repository. At the moment, you seem to be the best fit to hopefully finally bring this to a mergable state!

**Comment by upCASE** [comment]
Great to see some action here again! I had a look at both PRs and they look fine. I'll see if I can tests the changes today and merge them.

**Comment by iRet** [comment]
@upCASE thank you.
Please don't rush I might be able to polish this up a little bit more tonight.

The default input function is taken from Component and uses [this function](https://github.com/mixxxdj/mixxx/blob/main/res/controllers/midi-components-0.0.js#L86) to set value.
The group is referenced as this.group and I assume determined by the active deck.
While input function itself accepts the group from a midi message,  `inSetParameter` doesn't.
So the best approach I found is to override input function.

**Comment by upCASE** [comment]
Hi iRet,

had a very quick look at this.
Initial Filter knob value is okay, but for volume, tempo and EQ I don't see a change.

**Comment by iRet** [comment]
@upCASE did you test it on 2.2 or 2.3?
I use latest 2.3 beta.

File: res/controllers/Roland_DJ-202.midi.xml [hunk 1]
```diff
@@ -2289,15 +2289,6 @@
                     <script-binding/>
                 </options>
             </control>
-            <control>
```
Comment (iRet) [inline res/controllers/Roland_DJ-202.midi.xml:2292]: This was duplicated

**Comment by upCASE** [comment]
@iRet I used the latest 2.3 download on Windows 10. Removing the duplicated entry didn't change a thing.

**Comment by iRet** [comment]
@upCASE Surely it doesn't, just left a note for reviewer.
Most likely you are a doing something wrong.
I have just installed mixxx on windows to test this (not intended to use it there though)

Downloaded mapping files:
[Roland_DJ-202-scripts.js](https://github.com/Lykos153/mixxx/raw/fe0928ce5a24ae650435c6546293c887c291b506/res/controllers/Roland_DJ-202-scripts.js)
[Roland_DJ-202.midi.xml](https://github.com/Lykos153/mixxx/raw/fe0928ce5a24ae650435c6546293c887c291b506/res/controllers/Roland_DJ-202.midi.xml)
And put them to `C:\Users\<username>\AppData\Mixxx\controllers`

Everything works as good as in Linux.
I have recorded a video proof :) 
https://youtu.be/ow61NrxvYIE

**Comment by upCASE** [comment]
@iRet: You're right! Maybe I messed up and had the wrong file versions... Odd though, I'm pretty sure I copied them before.
Anyway: works fine, thanks heaps!

**Comment by upCASE** [comment]
@iRet : One small thing -> should the crossfader also work? This still has the "wrong" value after start.

**Comment by iRet** [comment]
There is a [note in 505 mapping](https://github.com/mixxxdj/mixxx/blob/main/res/controllers/Roland_DJ-505-scripts.js#L40) about this, according to which it doesn't send values within the parameter dump. Author stating it is firmware issue and could be fixed in future. Personally, I didn't check the crossfader handler yet.

**Comment by iRet** [comment]
@upCASE similar handler did the trick for cossfader too!

**Comment by upCASE** [comment]
Hi @iRet ,
works charming! Thanks heaps for this, I never found the time to figure this out!

IMHO this can be merged now, or do you plan further updates?

**Comment by iRet** [comment]
I don't have any objections if you have it tested.

The only thing I can think of is FX knobs, but they need to be selected with press and hold first for work.
Given thins it doesn't make any sense to read init values from them.

**Pull request merged**